pkg/api/v1alpha1/cluster_types.go

```diff
@@ -54,6 +54,13 @@ type ClusterSpec struct {
 	// Example: fs.aio-max-nr=232323
 	Sysctls    []string `json:"sysctls,omitempty"`
 	ScyllaArgs string   `json:"scyllaArgs,omitempty"`
+
+	// Timezone property links /etc/localtime on scylla containers to the host timezone file.
+	// Host timezone file path is calculated of Timezone value,
+	//   if Timezone starts with '/' it is assumed to be full path the the timezone file.
```

Comment:
```suggestion
	// if Timezone starts with '/' it is assumed to be full path the the timezone file.
```

Reply:
Some examples in comment section would be great

Reply:
added